test/DotNetEnvTraverse.Tests/TraverseTests.cs

```diff
@@ -1,7 +1,8 @@
 using System;
 using System.Linq;
 using Xunit;
-using DotNetEnv;  // only needed for ToDictionary extension method
+using DotNetEnv;
+using DotNetEnv.Extensions; // only needed for ToDictionary extension method

```

Comment:
Why are both `using`s needed now?

Reply:
just did not see that ;)

removed all unnecessary usings I found in the solution.